models/github.go

```diff
@@ -26,7 +26,7 @@ func (m DependencyGraphManifest) RepoURLFilename() string {
 
 // Ecosystem returns a name of ecosystem(or package manager) of manifest(lock) file in trivy way
 // https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/about-the-dependency-graph#supported-package-ecosystems
-func (m DependencyGraphManifest) Ecosystem() string {
+func (m DependencyGraphManifest) Ecosystem() ftypes.LangType {
```

Comment:
Regarding GitHub Dependency, do you think it is necessary to import the Trivy module?

Reply:
fyi https://github.com/package-url/packageurl-go/blob/fe183c1943ec36f257fae7143e160978217104b6/packageurl.go#L55-L114